apps/docs/src/app/core/component-docs/object-marker/object-marker-docs.component.html

```diff
@@ -0,0 +1,46 @@
+<fd-docs-section-title id="icon-only" componentName="objectmarker">
+    Object marker Only Icon Example
+</fd-docs-section-title>
+<description>
+    Standard list of icon used of the Object marker use <code>glyph</code> property to add icon.
+</description>
+<component-example>
+    <fd-object-marker-example></fd-object-marker-example>
+</component-example>
+<code-example [exampleFiles]="objectMarkerIconOnly"></code-example>
+<separator></separator>
+
+<fd-docs-section-title id="Object-marker-text" componentName="objectmarker">
+    Object marker Text and clickable Text
+</fd-docs-section-title>
+<description>
+    Standard Object marker Text and clickable use <code>clickable</code> boolean property to add hoverig effects.
+</description>
+<component-example>
+    <fd-object-marker-text-example></fd-object-marker-text-example>
+</component-example>
+<code-example [exampleFiles]="objectMarkerText"></code-example>
+<separator></separator>
+
+<fd-docs-section-title id="icon-text" componentName="objectmarker">
+    Object marker Icon and text readonly.
+</fd-docs-section-title>
+<description> Standard Object marker with icon and text use <code>glyph</code> property to add icon. </description>
+<component-example>
+    <fd-object-marker-icon-text-example></fd-object-marker-icon-text-example>
+</component-example>
+<code-example [exampleFiles]="objectMarkerIconandText"></code-example>
+<separator></separator>
+
+<fd-docs-section-title id="clickable" componentName="objectmarker">
+    Object marker Icon and text clickable.
+</fd-docs-section-title>
+<description>
+    Standard Object marker with icon and text with clickable use <code>clickable</code> boolean property to add hoverig
+    effects.
+</description>
+<component-example>
+    <fd-object-marker-clickable-example></fd-object-marker-clickable-example>
+</component-example>
+<code-example [exampleFiles]="objectMarkerClickable"></code-example>
```

Comment:
Could you add typescript file to example files? Now it doesn't support stackblitz